source/documentation/virtual_machine_management_guide/chap-Additional_Configuration.adoc

```diff
@@ -117,4 +117,5 @@ include::topics/Configuring_high_performance_virtual_machines.adoc[leveloffset=+
 include::topics/Configuring_Time_Zones.adoc[leveloffset=+1]
 
 // Very basic attribute, since I assume this section could one day be its own assembly and this attribute will therefore not be needed.
-:vmm:
+//:vmm:  this attribute seems to be messing up the compilation - testing commenting it out - Eli Marcus
```

Comment:
This was not causing the issue; Said added a step 3 with an example to the Configuring_Time_Zones.adoc and forgot to include the closing ---- for the example which read everything following it as part of the example. I have corrected the issue in https://github.com/oVirt/ovirt-site/pull/2679 and assigned you as a reviewer.